.github/workflows/ci.yml

```
@@ -1,21 +1,31 @@
-name: 'Tests'
+name: Braintree Node Example
 
-on: [push]
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
 
 jobs:
-  build:
-    name: 'Tests'
+  test:
+
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ['16', '18']
 
     steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js
-        uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - name: Set up Node
+        uses: actions/setup-node@v3
         with:
-          node-version: '14.x'
-      - run: npm install
-      - run: npm run test:unit
-      - run: npm start & sleep 3 && npm run test:integration
+          node-version: ${{ matrix.node-version }}
+
+      - run: npm ci
+
+      - run: npm run test
```

```suggestion
      - run: npm ci
      - run: npm run test:unit
      - run: npm start & sleep 3 && npm run test:integration
```

Need to do it this way so the test server starts up.

---

ahhhhhhhhhhhh